### PR TITLE
Added return-path documentation

### PR DESCRIPTION
**Pull request by maca134**
This is the documentation for https://github.com/fuel/email/pull/20

### CODE REVIEW

File: packages/email/introduction.html [hunk 1]
```diff
@@ -304,6 +304,14 @@ <h4 id="group_config">Group configuration</h4>
 								Leave the empty ('') in there to allow absolute paths.
 							</td>
 						</tr>
+                                                <tr>
```
Comment (frankdejonge) [inline packages/email/introduction.html]: indenting is messed up here as well

File: packages/email/methods.html [hunk 2]
```diff
@@ -1168,6 +1168,48 @@ <h4 class="method" id="method_get_invalid_addresses">get_invalid_addresses()</h4
 					</tbody>
 				</table>
 			</article>
+                        
+                        <article>
```
Comment (frankdejonge) [inline packages/email/methods.html]: same here

Comment (maca134) [inline packages/email/methods.html]: Its my IDE, looks fine in Netbeans, I will sort it now

**Comment by maca134** [comment]
Sorted

**Pull request merged**